### ISSUE TITLE
Some of the problems that occur in training

### ISSUE THREAD

**Issue by Dyongh613** [keonlee9420/DiffGAN-TTS#15]
Hi@keonlee9420, I encountered some problems during the training stage. I often have loss functions that occasionally fluctuate a lot during training, even from around 3 to tens or hundreds. After I set the training set shuffle, sometimes I have this problem, sometimes but not this problem. This problem was encountered in the naive, aux and shallow stages. Thank you for my friend！Best wish to you!

**Comment by Dyongh613** [comment]
By the way, it occurs on D_loss and adv_loss
